cmd/timestamp-server/app/serve.go

```diff
@@ -100,8 +100,8 @@ var serveCmd = &cobra.Command{
 		host := viper.GetString("host")
 		port := int(viper.GetUint("port"))
 		scheme := viper.GetStringSlice("scheme")
-
-		server := server.NewRestAPIServer(host, port, scheme, readTimeout, writeTimeout)
+		log.Logger.Debugf("httpPingOnly: %t", httpPingOnly)
```

Comment:
did you mean to leave this in?

Reply:
as a debug message,  I think it can be useful -  similar to the one on Line 68 about `proof`.  Here's how it would look for most usages:
```
$ timestamp-server serve
2023-09-27T22:57:49.371+0200	INFO	app/serve.go:52	starting timestamp-server @ {
  "gitVersion": "devel",
  "gitCommit": "0e597412ed9a544075bd279bd09c7c2f3b7dc529",
  "gitTreeState": "clean",
  "buildDate": "2023-09-27T20:55:25",
  "goVersion": "go1.21.1",
  "compiler": "gc",
  "platform": "darwin/arm64"
}
2023-09-27T22:57:49.372+0200	DEBUG	app/serve.go:68	pprof enabled: false
2023-09-27T22:57:49.372+0200	DEBUG	app/serve.go:103	httpPingOnly: false
2023-09-27T22:57:49.373+0200	INFO	ntpmonitor/ntpmonitor.go:154	ntp monitoring starting
2023-09-27T22:57:49.373+0200	DEBUG	ntpmonitor/ntpmonitor.go:192	querying ntp server sth1.ntp.se
```
Would you prefer to remove it?

Reply:
removed in b09fff4